src/star-rating.jsx

```diff
@@ -83,43 +83,50 @@ export default class StarRating extends React.Component {
         width: "15px",
         fontSize: "32px",
         overflow: "hidden",
-      }
+      };
 
       stars.push(
-        <span
-          className="star" key={`star-${i+0.5}`}
-          name={name}
-        >
+        <span className="star" key={`star-${i + 0.5}`} name={name}>
           <span
             style={{
               ...starStyles,
-              direction: "ltr"
+              direction: "ltr",
             }}
             className={starClass}
-            onMouseOver={() => this.handleMouseover(i+0.5)}
-            onClick={() => this.rate(i+0.5)}
+            onMouseOver={() => this.handleMouseover(i + 0.5)}
+            onClick={() => this.rate(i + 0.5)}
             onMouseOut={() => this.handleMouseout()}
-          >{isValidRating ? <i className="fa fa-star" aria-hidden="true"></i> : <i className="fa fa-star-o" aria-hidden="true"></i>}
+          >
+            {isValidRating ? (
+              <i className="fa fa-star" aria-hidden="true"></i>
+            ) : (
+              <i className="fa fa-star-o" aria-hidden="true"></i>
+            )}
           </span>
           <span
             style={{
               ...starStyles,
-              direction: "rtl"
+              direction: "rtl",
             }}
             className={starClass}
-            onMouseOver={() => this.handleMouseover(i+1)}
-            onClick={() => this.rate(i+1)}
+            onMouseOver={() => this.handleMouseover(i + 1)}
+            onClick={() => this.rate(i + 1)}
             onMouseOut={() => this.handleMouseout()}
-          >{rating >= i+1 && rating !== 0 ? <i className="fa fa-star" aria-hidden="true"></i> : <i className="fa fa-star-o" aria-hidden="true"></i>}
+          >
+            {rating >= i + 1 && rating !== 0 ? (
+              <i className="fa fa-star" aria-hidden="true"></i>
+            ) : (
+              <i className="fa fa-star-o" aria-hidden="true"></i>
+            )}
           </span>
         </span>
       );
     }
     return (
       <div className="rating-stars">
-        <div className="star-container">
-        {stars}
-        <span className="rating-number">{rating}</span>
+        <div className={`star-container ${disabled ? "star-disabled" : ""}`}>
```

Comment:
@VijayendrenR It adds style for disabled input. 

But, we also need the privilege to pass class name prop to star rating input.